department-of-veterans-affairs/caseflow
# Adding DIC to appeal attributes

Connects #4337 

To test:
1. Find an issue in the database that's DIC 
1. Create a hearing for that issue's appeal
1. Go to the hearing worksheet and confirm that it's displayed as DIC
